Worn Out/Worn Out/Models/Run.swift

```diff
@@ -0,0 +1,16 @@
+//
+//  Run.swift
+//  Worn Out
+//
+//  Created by Vici Shaweddy on 2/27/20.
+//  Copyright © 2020 Vici Shaweddy. All rights reserved.
+//
+
+import Foundation
+
+struct Run {
+    let workoutIdentifier: String //HKObject UUID
```

Comment:
Look up typealias - you could potentially use it here if you want to define a HealthKitUUID as a String, so it is clear *what* kind of string this is.

Reply:
Thanks for the suggestion! I will look it up!